src/commands/list.ts

```diff
@@ -43,7 +43,7 @@ export function listCommand(program: Command) {
         };
       });
 
-      logger.info(wrapTable(tableData));
+      logger.info(wrapTable(tableData) as string);
```

Comment:
This function doesn't return a string. So... you should be fixing it someplace (probably in wrapTable, or maybe logger.info can in fact log non-strings?). Not casting it

Edit: you set string in `wrapTable` later on, so I think these casts are probably unnecessary